packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -67,6 +66,45 @@ import {
 } from "./BlockNoteTipTapEditor";
 import "./editor.css";
 
+export function checkImageInSchema<
```

Comment:
if these functions are used in other files as well, we should not define them in this file